CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Guide to Making Developer Contributions
+
+## Overview:
+
+A CSV with package name and version is uploaded by the user, 
+then the risk of the packages are assessed using `riskmetric`.
+[TODO this is shakey for me.... where is this function actually called... 
+I see where we write comments to the database, 
+and where we read fields FROM the database,
+but where and how are the database's risk metrics written?]
+
+The database (SQLite by default) is updated to include the user specified package 
```

Comment:
Getting rid of the SQLite db. Robert has an open PR to do this, so watch out for the merge =)